soc/aesc/vexriscv/soc.yml

```diff
@@ -0,0 +1,6 @@
+family:
+- name: aesc_vexriscv
```

Comment:
`aesc_vexriscv` doesn't sound like SoC family. It's just the vendor name + CPU name. Please note that the `family` property is not required.

Reply:
I didn't know that. Thanks! Removed the `family` property and replaced `vexriscv` with the  `nitrogen` directory.